src/api/__tests__/api.unit.test.ts

```diff
@@ -1,81 +1,351 @@
 import "core-js";
 import mockAxios from "jest-mock-axios";
-import { API_ROOT, getApiUrl, apiGet } from "../api";
-import ERRORS from "../../shared/errors";
-import { makeTestError } from "../../shared/test-utils";
+import {
+  retrieveFactory,
+  getFactory,
+  createFactory,
+  patchFactory,
+  deleteFactory,
+  endpointFactory,
+  unwrap,
+} from "../api";
+import {
+  MOCK_RESPONSE,
+  MOCK_ERROR,
+  MOCK_BODY,
+  MOCK_CSRF,
+  MOCK_PARAMS,
+} from "../../shared/test-utils";
 
-describe("getApiUrl", () => {
-  it("returns root endpoint on empty string", () => {
-    expect(getApiUrl("")).toBe(API_ROOT);
+describe("listFactory", () => {
```

Comment:
Missing test cases with paginated data

Reply:
So, my opinion on these tests, as well as the other comments on the `useAPILazy` tests, is that these hooks really just forward data, and there's really no separate functionality to test with paginated data, it just gets passed through like any other kind of data.